shepherd/Utils.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum, unique
+
+class Header(Enum):
+    # Used to autoenumerate
+    def __new__(cls):
+        value = len(cls.__members__) + 1 # pylint: disable=no-member
+        obj = object.__new__(cls)
+        obj._value_ = value # pylint: disable=protected-access
+        return obj
+
+@unique # pylint: disable=invalid-name
+class SHEPHERD_HEADER(Header):
```

Comment:
It seems like there should be some sort of documentation associated with these headers... again tho, if it's just for us then it doesn't really matter.